etc/helm/pachyderm/values.yaml

```diff
@@ -523,7 +523,9 @@ pachd:
   #   - repoReader
   #   robot:eve:
   #   - repoWriter
-  pachAuthClusterRoleBindings: {}
+  pachAuthClusterRoleBindings:
+    allClusterUsers:
+      - projectCreator # by default, everyone should be able to create projects
```

Comment:
No need for projectOwner?

Reply:
No need for projectOwner. We want users to create projects, and become owners, but don't want to automatically make everyone owner by default.